ql/src/java/org/apache/hadoop/hive/ql/Driver.java

```diff
@@ -366,13 +367,22 @@ private void acquireLocks() throws CommandProcessorException {
         driverContext.getTxnManager().getTableWriteId(t.getDbName(), t.getTableName());
       }
 
-
       DDLDescWithWriteId acidDdlDesc = driverContext.getPlan().getAcidDdlDesc();
       boolean hasAcidDdl = acidDdlDesc != null && acidDdlDesc.mayNeedWriteId();
+      LOG.info(" acidDdlDesc : " + acidDdlDesc );
+      if ( acidDdlDesc != null )
+        LOG.info("  mayNeedWriteId : " + acidDdlDesc.mayNeedWriteId());
```

Comment:
Please review the log levels. I think it is too much on INFO level. Please reduce the lines, and maybe move some stuff to DEBUG

Reply:
Added some for debugging. Will remove all of those in the next patch.